WebApplication1/Services/BookServices.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using WebApplication1.BookData;
+
+namespace WebApplication1.Services
+{
+
+    public class BookServices

```

Comment:
Class name must be BookService as a single service can have multiple functionalities